src/test/Components/CreateImageWizard/CreateImageWizard.compliance.test.js

```diff
@@ -84,36 +87,73 @@ describe('Step Compliance', () => {
     await clickNext();
 
     // Now we should be in the Compliance step
-    await screen.findByRole('heading', { name: /OpenSCAP Compliance/i });
-    expect(
-      await screen.findByRole('radio', { name: /do not add a profile/i })
-    ).toBeChecked();
+    await screen.findByRole('heading', { name: /OpenSCAP/i });
+
     await user.click(
-      await screen.findByRole('radio', { name: 'Add a profile' })
+      screen.getByRole('textbox', { name: /select a profile/i })
     );
-    expect(
-      await screen.findByRole('radio', { name: 'Add a profile' })
-    ).toBeChecked();
+    await user.click(screen.getByText(/none/i));
+
+    // check that the FSC does not contain a /tmp partition
+    await clickNext();
+    await screen.findByRole('heading', { name: /File system configuration/i });
+    await screen.findByRole('cell', {
+      name: /xfs/i,
+    });
```

Comment:
I see, in case there would be more partitions the test would fail as there would be multiple cells with "xfs" as a content. I don't really mind it, but it might not be immediately obvious. What you can do is simply query for the tmp mount point and check it's not there when everything renders.

Which can be done like this:

```suggestion
    expect(
      screen.queryByRole('cell', {
        name: /tmp/i,
      })
    ).not.toBeInTheDocument();
```